lib/api/include/irods/get_resource_info_for_operation.h

```diff
@@ -0,0 +1,38 @@
+#ifndef IRODS_GET_RESOURCE_INFO_FOR_OPERATION_H
+#define IRODS_GET_RESOURCE_INFO_FOR_OPERATION_H
+/// \file
+
+#include "irods/dataObjInpOut.h"
+
+struct RcComm;
+
+/// Get preferred-resource information for the specified operation.
+///
+/// \parblock
+/// \param[in] _conn - An rcComm_t connection handle to the server.
+/// \param[in] _dataObjInp - generic dataObj input. Relevant items are:
```

Comment:
Use `\parblock` and `\endparblock` to define a paragraph or group related text.

For example:
```cpp
/// \param[in] _dataObjInp \parblock A DataObjInp describing the data object operation.
///
/// Relevant members of input structure are:
/// - \p objPath: The logical path of the target data object.
/// - \p condInput: A list of options that influence the results of the operation.
///     - \p REPL_NUM_KW: An integer identifying a specific replica of the data object.
///     - \p GET_RESOURCE_INFO_FOR_OPERATION_KW: The operation of interest. The following values are supported:
///         - "CREATE"
///         - "WRITE"
///         - "OPEN"
///         - "UNLINK"
///     - \p RESC_NAME_KW: The name of a resource. Only used to create a new file, no overwrite of existing files.
/// \endparblock
```

`\p` causes the next token to be formatted differently.